teachopencadd/talktorials/019_One_Hot_Encoding/talktorial.ipynb

```diff
@@ -0,0 +1,2926 @@
+{
```

Comment:
<p>I suggest format the reference as:</p><p><code>- ChEMBL database: "The ChEMBL bioactivity database: an update." ([&lt;i&gt;Nucleic acids research&lt;i&gt; (2014), &lt;b&gt;42.D1&lt;/b&gt;, D1083-D1090](https://doi.org/10.1093/nar/gkt1031))</code></p><p> instead of pasting the URL directly.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/TeachOpenCADD/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Done.